charts/kubernetes/templates/playbooks/gitops-request-namespace-access.yaml

```diff
@@ -0,0 +1,111 @@
+{{- if and .Values.playbooks .Values.playbooks.edit_kubernetes_manifests.enabled }}
+---
+apiVersion: mission-control.flanksource.com/v1
+kind: Playbook
+metadata:
+  name: request-namespace-access-gitops
+spec:
+  configs:
+    - types:
+        - Kubernetes::Namespace
+      labelSelector: 'kustomize.toolkit.fluxcd.io/name' # limits to namespaces created by flux
+  title: Request Namespace Access (GitOps)
+  category: Kubernetes
+  icon: add-shield
+  description: Adds a Rolebinding to the given role in the selected Namespace
+  env:
+    - name: file_path
+      value: {{ "'{{" }} (.config.config | jq `.metadata.annotations["config.kubernetes.io/origin"]` | yaml).path {{ "}}'" }}
+    - name: git_url
+      value: {{ "'{{" }} (index (catalog_traverse .config.id "Kubernetes::Kustomization/Kubernetes::GitRepository" "incoming") 0).Config | json | jq `.spec.url` {{ "}}'" }}
+    - name: git_branch
+      value: {{ "'{{" }} (index (catalog_traverse .config.id "Kubernetes::Kustomization/Kubernetes::GitRepository" "incoming") 0).Config | json | jq `.spec.ref.branch` {{ "}}'" }}
+    - name: kustomization_path
+      value: {{ "'{{" }} (index (catalog_traverse .config.id "Kubernetes::Kustomization" "incoming") 0).Config | json | jq `.spec.path` {{ "}}'" }}
+  parameters:
+    - name: connection
+      label: Repository Auth connection
+      default: "{{.Values.playbooks.edit_kubernetes_manifests.git_connection}}"
+    - name: user
+      label: Role Subject User
```

Comment:
Default to the current user email

Reply:
https://github.com/flanksource/mission-control/compare/feat/playbook-user-template-on-run?expand=1